thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -70,8 +70,8 @@ fi
 
 if [ ! -f nunit.framework.dll ]; then
 	echo "Fetching NUnit from NuGet"
-	get NUnit 3.0.1
-	cp ./NUnit/lib/net40/nunit.framework* .
+	get NUnit 3.4.1
```

Comment:
`NUnit.Console` has also been updated to 3.4.1, so I guess we'll want to keep these in sync.